### ISSUE TITLE
RAM Allocation cannot go above 8140 even with "allow full ram allocation" toggled on

### ISSUE THREAD

**Issue by KittyFloof** [FTBTeam/FTB-App#1046]
### What Operating System

Windows 10

### Debug Code

FTB-DBGZIDIFOHORO

### Describe the bug

Even when instance is unlocked and "allow full ram allocation" is applied there is no way to use more than 8140 it seems like the toggle does absolutely nothing and does not save any settings applied. I want to use more ram but there is no way to actually do so. 

### Steps to reproduce

1. go to the settings for an instance
2. try to move the RAM Allocation slider to the end
3. find out your locked to only 8140 (7.7gb)
4. toggle "allow full ram allocation" 
5. find out it does nothing
6. leave the options menu and come back
7. toggle turns off and ram is still locked to max 8140 (7.7gb)
8. enter a valid value 10240 (10gb) with "allow full ram allocation" on
9. leave options end come back
10. returns to 8140 (7.7gb)
11. unlock instance and do all of the above again
12. nothing changes

### Expected behaviour

being allowed to use as much RAM as I please

**Comment by MichaelHillcox** [comment]
How much ram does your system have?

**Comment by KittyFloof** [comment]
> How much ram does your system have?

16g if I go into the config files for the instance I can force the change but that's annoying to have to do.

**Comment by MichaelHillcox** [comment]
From the debug report you attached, it's reporting that you only have 8gb of ram which is... weird...

**Comment by KittyFloof** [comment]
> From the debug report you attached, it's reporting that you only have 8gb of ram which is... weird...

Running cpuz says otherwise, windows also says otherwise... Maybe my ram is dying for whatever reason? But I doubt that though. Perhaps something is wrong with the check for it? I have 2 sticks installed and everything I see is telling me I have 16g.

**Comment by KittyFloof** [comment]
haha so funny thing.... my ram was in fact dead. barely hanging on by a thread apparently. I looked deeper into this and ran memtest and wow the number of errors it found. so you were correct in that the debug code was recording 8 gb because that was all it could use! So I got myself and early gift and bought new ram. This issue was entirely my fault. Sorry for making a ticket about this, I truly thought there was something wrong with the app. happy holidays and sorry for wasting your time.
